scripts/qa_crowbarsetup.sh

```diff
@@ -297,6 +297,7 @@ function wait_for()
     do
         echo -n .
         sleep $timesleep
+        [[ $(($n % 75)) != 0 ]] || echo
```

Comment:
this renders the same unexpected output :)

This is what you are looking for:
`
[[ $(( $(( $timecount - $n + 1 )) % 75)) != 0 ]] || echo
`